scripts/get-data.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import pandas as pd
+
+DATASET_FIELDS = [
```

Comment:
Can you add the following parameters- 
- projectOutComesReport
- piemail
- publicationResearch
- publicationConference
- startDate
- expDate

Reply:
> 

Keeping them in the repo is perfectly fine